RELEASE-NOTES.txt

```diff
@@ -12,6 +12,7 @@
 * [*] Block editor: Add missing translations of unsupported block editor modal [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4410]
 * [**] Time zone suggester: we have a new time zone selection screen that suggests the time zone based on the device, and improves search. [#17699]
 * [*] Added the "Share WordPress with a friend" row back to the Me screen. [#17748]
+* [***] Updated default app icon. [#17793]
```

Comment:
Sadly, those won't make it in the Release Notes shown in the AppStore because those have already gone thru editorial/wordsmithing and into the GlotPress pipeline for translations.

We could maybe consider trying to fit that new entry in the [English, editorialized release notes](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/WordPress/Resources/release_notes.txt) somehow (if someone how has English writing skills want to give it a try), but those won't be translated in time in other locales anyway so even if we decide to go that route, that new item would only appear in the English AppStore.

Reply:
No worries, I didn't expect they would. I just wanted to list it down for future reference.